src/components/Patient/PatientRegister.tsx

```diff
@@ -1022,6 +1026,7 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                       {...field("name")}
                       type="text"
                       label={"Name"}
+                      autoCapitalize="words"
```

Comment:
Off here; we accept whatever the user entered.

Reply:
@Jacobjeevan do u mean , 
write
autoCapitalize = "off" 
?

Reply:
yes

Reply:
@Jacobjeevan we decided to hint mobile keyboards to auto capitalize right? User can still override it.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize

<img width="787" alt="image" src="https://github.com/user-attachments/assets/38464b2a-391f-4316-84de-411020ebf4a7">

Reply:
> https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize
> <img alt="image" width="787" src="https://private-user-images.githubusercontent.com/25143503/391211727-38464b2a-391f-4316-84de-411020ebf4a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5NDE0ODYsIm5iZiI6MTczMjk0MTE4NiwicGF0aCI6Ii8yNTE0MzUwMy8zOTEyMTE3MjctMzg0NjRiMmEtMzkxZi00MzE2LTg0ZGUtNDExMDIwZWJmNGE3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTMwVDA0MzMwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiMGVhZmJjNWM5MWU2MzQ3NjZlZjM3Njk4MWNhMWIwZTkyNDA3OTg3YjFiZmM4ZWM2M2E5ZjQ2MDZkYzBhNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Ce8xadLybyzE1fhywCtm_em22ZioaM7MltIOPXAKfKM">

Ah, ok no worries if users can override it; I thought it would keep it/pass it as capitalized.